tools/compiletest/src/runtest.rs

```diff
@@ -75,6 +77,7 @@ impl<'test> TestCx<'test> {
             KaniFixme => self.run_kani_test(),
             CargoKani => self.run_cargo_kani_test(false),
             CargoKaniTest => self.run_cargo_kani_test(true),
+            CoverageBased => self.run_expected_coverage_test(),
```

Comment:
We could, but we thought it would be better to strongly enforce the flag for this suite, since the testing is sensitive to the flag being present throughout the suite. If it's not a problem, I think it makes sense to add a new mode for now and change the logic to not use expected files in a future PR.